### PR TITLE
Grid width styles revert temp fix, now coming in from Milo

### PR DESCRIPTION
**Pull request by ryanmparrish**
There are a few pages on Bacom that reported some display issues after [recent updates](https://github.com/adobecom/milo/commit/55afc5c6ab3e8f19b2fb8c2bfc9858341e0e6104#diff-7d6d5811a084df948c503699455ce384eb2f73adec6f11a4ad201c74c4ecf16dR175-R180%23diff-7d6d5811a084df948c503699455ce384eb2f73adec6f11a4ad201c74c4ecf16dR175) to the section metadata styles: grid-width-*

This PR reverts the style patch [pr](https://github.com/adobecom/bacom/pull/151) that was in place since these are now working as expected from milo. 

Resolves: [MWPW-137076](https://jira.corp.adobe.com/browse/MWPW-137076)

Test URLs:

LIVE Grid Width testing
Current - bacom fix in place - Should display as expected
https://main--bacom--adobecom.hlx.live/products/experience-manager/digital-commerce

Before - bacom fix reverted - Should break grid
https://rparrish-grid-width--bacom--adobecom.hlx.live/products/experience-manager/digital-commerce

After - bacom fix reverted && milo feature branch used - Should display as current/expected
https://rparrish-grid-width--bacom--adobecom.hlx.live/products/experience-manager/digital-commerce?milolibs=rparrish-grid-width

LIVE Charts
Current
https://main--bacom--adobecom.hlx.live/resources/digital-price-index
Before Fix
https://rparrish-grid-width--bacom--adobecom.hlx.live/resources/digital-price-index
After
https://rparrish-grid-width--bacom--adobecom.hlx.live/resources/digital-price-index?milolibs=rparrish-grid-width

Draft Charts
Current
https://main--bacom--adobecom.hlx.live/drafts/parrish/resources/digital-price-index
Before Fix
https://rparrish-grid-width--bacom--adobecom.hlx.live/drafts/parrish/resources/digital-price-index
After
https://rparrish-grid-width--bacom--adobecom.hlx.live/drafts/parrish/resources/digital-price-index?milolibs=rparrish-grid-width

### CODE REVIEW

**Comment by meganthecoder** [comment]
@ryanmparrish I see that this is a draft. Is this ready to review or are you waiting until the milo PR is merged?

**Comment by ryanmparrish** [comment]
@meganthecoder  && @JasonHowellSlavin - Sorry for the delay on this. I am ready to merge the related milo PR and then sequentially merge this one. Just getting some eyes on this. Hoping to do this today. LMK if anyone is available to coordinate these efforts.

**Review by meganthecoder — APPROVED** [review]

**Comment by meganthecoder** [comment]
@ryanmparrish Can you update your bacom urls to use hlx.live? main--bacom--adobecom.hlx.page is behind the vpn so performance testing doesn't work

**Review by JasonHowellSlavin — APPROVED** [review]

**Pull request merged**
